README.md

```diff
@@ -1 +1,55 @@
-# React-Capstone
\ No newline at end of file
+# Covid Metrics Webapp for Spain
+> This is a single page application (SPA) that shows current covid stats for Spain. It consumes the [Narrativa API](https://covid19tracking.narrativa.com/index_en.html). Users can use the filters to see regions with no new cases or regions with the highest number of cases for the day.
+
+## Screenshots
+
+Homepage:
+  ![Screen Shot 2021-10-15 at 10 23 19](https://user-images.githubusercontent.com/10905837/137460007-ee391052-62f3-44d2-8013-533f417901ee.png)
+  Details Page:
+![Screen Shot 2021-10-15 at 10 51 02](https://user-images.githubusercontent.com/10905837/137460320-f022385b-8e55-4068-bcba-9438086942f0.png)
+
+
+[Loom video presentation](https://www.loom.com/share/67630cd92fbe48f28395f79ffe7d0bae)
+
+## Built With
+
+- Javascript
+- React
+- Redux
+- Jest
+- Bootstrap
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://relaxed-jang-026dde.netlify.app/)
+
+## Getting Started
+
+### To get a local copy run the following steps:
+
+- Open your terminal or command line and run `git clone https://github.com/themonster2015/React-Capstone.git`
+- Open the folder with your code editor (VS Code for example)
+- Run `npm install`
+- Run `npm run build`
+- Run `npm start` to launch the server and view it in the browser.
+
+## Author
+
+👤 **Vo Thi Quynh Yen**
+
+- GitHub: [@themonster2015](https://github.com/themonster2015)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/themonster2015/React-Capstone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
```

Comment:
- [ ] Kindly give credit to Nelson Sakwa, Creator of the original design as per the project requirements.